woocommerce/woocommerce-gutenberg-products-block
# ProductControl: Update product search performance

This brings over the search performance improvements in the `ProductsControl` into `ProductControl` – see #515 for background.

### How to test the changes in this Pull Request:

Use the Featured Product block to test this, as it uses ProductControl to pick a product. Keep the network tab of your dev tools open, so you can watch the API calls.

If your store has less than 200 products, you'll only see the API calls when the ProductControl loads – there could be 2 as it automatically pages through 1-100, 101-200. Searching the list should be quick, and not trigger any other API requests.

If your store has over 200 products, you'll see one API call when the ProductControl loads, for just the first page of products. Each search will also trigger an API request for your search term.

In any case, you should be able to find the products you're searching for 👀
